bin/alluxio-mount.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$-" == *x* ]]; then
 fi
 BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
 
-USAGE="Usage: alluxio-mount.sh [Mount|SudoMount] [MACHINE]
+USAGE="Usage: alluxio-mount.sh --umount [Mount|SudoMount|Umount|SudoUmount] [MACHINE]
```

Comment:
Why adding `--umount` here?
Would it be better to use `alluxio-mount.sh [Mount|SudoMount|Umount|SudoUmount] [MACHINE]` directly?

Reply:
good catch, I was originally thinking of using `--umount`, but then changed it. I will delete `--umount`